src/main/app/src/tools/utils.js

```diff
@@ -54,24 +54,28 @@ export const getSearchAddress = (postitoimipaikka = '', osoite = '') => {
   };
 };
 
-export function formatDateString(dateString, dateFormat = 'd.M.y HH:mm') {
+export function formatDateString(dateString) {
   if (!dateString) {
     return '';
   }
 
-  const klo = i18n.t('lomake.klo');
-
-  if (klo !== '') {
-    dateFormat = "d.M.y 'klo' HH:mm";
+  const date = new Date(dateString);
+  switch (getLanguage()) {
+    case 'fi':
+      return format(date, "d.M.y 'klo' HH:mm");
+    case 'en':
+      return formatToTimeZone(date, 'd MMMM YYYY h:mm A z', {
+        timeZone: 'Europe/Helsinki',
+      });
+    case 'sv':
+      return format(date, "d.M.y 'kl.' HH:mm");
+    default:
+      return format(date, "d.M.y 'klo' HH:mm");
```

Comment:
"fi" ja default caset on samat, voi yhdistää. `case 'fi':` on kuitenkin hyvä selkeyden vuoksi mun mielestä olla, vaikka menis pelkällä defaultilla.
```suggestion
    case 'en':
      return formatToTimeZone(date, 'd MMMM YYYY h:mm A z', {
        timeZone: 'Europe/Helsinki',
      });
    case 'sv':
      return format(date, "d.M.y 'kl.' HH:mm");
    case 'fi':
    default:
      return format(date, "d.M.y 'klo' HH:mm");
```

Reply:
Muutettu